quantlib/instruments/bonds.pyx

```diff
@@ -212,3 +216,53 @@ cdef class ZeroCouponBond(Bond):
                 'Wrapper for such constructor not yet implemented.'
             )
 
+cdef class FloatingRateBond(Bond): 
+    """ Floating Rate Bond """
+    def __init__(self, int settlement_days, double face_amount, Schedule float_schedule, 
+        IborIndex ibor_index, DayCounter accrual_day_counter, int fixing_days, 
+        gearings, spreads, caps, floors, payment_convention=Following, redemption=100.0, Date issue_date=None
+        ):
+        """Floating rate bond (Instantiation): 
```

Comment:
Another convention we push for is the numpydoc like docstrings (https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt)

That would suggest your docstring is formatted like this:

```
""" Floating rate bond constructor

Parameters
----------
settlement_days : int
    Number of days before the operation settles
face_amount: float (which is a C double in Python)
    Face amount of the bond
...
```